protected/models/SurveyLanguageSetting.php

```diff
@@ -92,6 +92,8 @@ public function relations()
     public function rules()
     {
         return [
+            ['surveyls_survey_id','required', 'on' => 'insert'],
+            ['surveyls_language','required', 'on' => 'insert'],
```

Comment:
... must be fixed : no ability to create SUrvey : $this->survey is null at create